ui/src/app/Deployment/DeploymentProgress/DeploymentProgress.tsx

```diff
@@ -14,33 +14,36 @@ interface IDeploymentProgressProps {
 export const DeploymentProgress = ({ progressData }: IDeploymentProgressProps ) => {
 
   // set a deployment message based on the status
-  let  deploymentMessage = ""
+  let deploymentMessage = ""
+  let cleanupMessage = "You still need to delete the managed application from your Azure subscription. " +
+  "For more information about deleting resources, refer to the documentation linked to the left."
   if (progressData.isComplete) {
     deploymentMessage = "Your Ansible Automation Platform deployment is now complete."
   } else if (progressData.isCanceled) {
-    deploymentMessage =
-      "Your Ansible on Azure deployment is cancelled. You still need to delete the managed application from your Azure subscription." +
-      "In the Azure Portal, navigate to 'Resource Groups', and then to the resource group where you deployed the instance of the managed application. " +
-      "Select the managed application from the list of resources and then click 'Delete' to remove all resources associated with the managed application"
+    deploymentMessage = "Your Ansible on Azure deployment is cancelled. "
```

Comment:
Instead of calling `.concat()` on the next line, we can inline it with use of backticks:
```js
deploymentMessage = `Your Ansible on Azure deployment is cancelled. ${cleanupMessage}`
```
Same can be done in following `else if` block.

Reply:
Cool, will do.